_posts/blog/2023-06-10-2022-okr-wrap.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+categories: 
+  - blog
+title: "CHN Board: 2022 OKR Review"
+description: ""
+date: 2023-06-10
+image: /images/logo/logo-star.png
+author: Sean Watland
+author_url: https://chihacknight.org/board-of-directors.html
+author_image: /images/logo/logo-star.png
+author_bio: 
+author2: 
+author2_url: 
+author2_image: 
+author2_bio: 
+published: true
+featured: false
+---
+
+__Blurb Summarizing how we did... to be run by board later on....__
+
+__blurb reflecting on successes / draw backs from this year...__
+
+### **Objective 1 - Run the event** {#objective-1-run-the-event}
+
+_Produce a live, weekly civic engagement event that meaningfully includes diverse speakers and attendees_
+
+<span class='label label-success'><i class='fa fa-fw fa-check'></i> ACHIEVED</span> Key Result 1.1: Run Chi Hack Night every week, excluding planned holiday weeks
+
+<span class='label label-success'><i class='fa fa-fw fa-check'></i> ACHIEVED</span> Key Result 1.2: Ensure each presenter represents at least one of Chi Hack Night’s values
+
+<span class='label label-success'><i class='fa fa-fw fa-check'></i> ACHIEVED</span> Key Result 1.3: Measure and track the diversity of the Chi Hack Night membership, presenters and co-hosts
+
+<span class='label label-success'><i class='fa fa-fw fa-check'></i> ACHIEVED</span> Key Result 1.4: Ensure that at least 30% of presenters are persons of color and at least 50% of presenters fill out the demographic survey
+
+<span class='label label-success'><i class='fa fa-fw fa-check'></i> ACHIEVED</span> Key Result 1.5: Ensure that at least 50% of co-hosts are from underrepresented groups in tech
+
+<span class='label label-warning'><i class='fa fa-fw fa-times'></i> PARTIALLY ACHIEVED</span> Key Result 1.6: Measure the satisfaction of participants that attend our hybrid and online events to make them more inclusive and engaging
+
+
+### **Objective 2 - Sustain the event** {#objective-2-sustain-the-event}
+
+_Build & maintain a sustainable infrastructure for weekly operations engaging volunteers and expanding our organization’s capacity_
+
+<span class='label label-success'><i class='fa fa-fw fa-check'></i> ACHIEVED</span> Key Result 2.1: Each quarter, each volunteer role has been performed by at least two different people
+
+<span class='label label-warning'><i class='fa fa-fw fa-times'></i> PARTIALLY ACHIEVED</span> Key Result 2.2: Recruit and onboard one new non-board co-host, greeter, “Welcome to CHN” host for operational functions at Chi Hack Night per quarter
+
```

Comment:
The [Google Doc](https://docs.google.com/document/d/10bsLrIG7XpG73AsuiW3WntzC26qs_gjWpauQe0VTJGA/edit) has another KR here about tracking when tasks are completed.

Reply:
my apologies for missing these! For the missing KRs, this is likely because I took the KRs from what we published in [the year in review ](https://chihacknight.org/blog/2023/02/07/2022-year-in-review.html). Since these are public facing, I'd opt to not change / add any of the olds ones at this time. 

I know it's messy, but these OKRs were a bit messy anyways. So I'd rather just keep the ones we have listed and move forward.